plugins/Denizen/scripts/restraining_order.dsc

```diff
@@ -0,0 +1,33 @@
+restraining_order:
+    type: item
+    display name: <&f>Restraining Order
+    material: paper
+    flags:
+        uuid: <util.random_uuid>
+    lore:
+        - <&7>Right click to assign to a player!
+
+restraining_order_handler:
+    type: world
+    debug: true
+    events:
+        on player right clicks entity with:restraining_order type:player:
+            - if <context.item.has_flag[restrained]>:
+                - stop
+            - define slot <player.held_item_slot>
+            - inventory flag slot:<[slot]> restrained:<context.entity.if_null[<empty>]>
+            - define item <player.item_in_hand>
+            - inventory adjust slot:<[slot]> lore:<&7>Attached<&sp>to<&sp><[item].flag[restrained].name>
```

Comment:
The lore can become:
`"lore:<&7> Attached to ....."` instead of having `<&sp>` to indicate a space.

Reply:
Slot should be `offhand`.